spec/requests/alert_definitions_spec.rb

```diff
@@ -81,7 +81,7 @@
     alert_definition = MiqAlert.find(response.parsed_body["results"].first["id"])
     expect(alert_definition).to be_truthy
     expect(alert_definition.expression.class).to eq(MiqExpression)
-    expect(alert_definition.expression.exp).to eq(sample_alert_definition["expression"])
+    expect(alert_definition.expression.exp).to eq(sample_alert_definition["expression"].deep_symbolize_keys)
```

Comment:
This looks wrong, because miq expressions are not keyed with symbols (see above)